vagrant_cloud.gemspec

```diff
@@ -13,7 +13,8 @@ Gem::Specification.new do |s|
   s.executables << 'vagrant_cloud'
 
   s.add_runtime_dependency 'rest-client', '~> 2.0.2'
-  s.add_runtime_dependency 'thor', '~> 0.19.4'
+  #s.add_runtime_dependency 'thor', '~> 0.19.4'
+  s.add_runtime_dependency 'json', '~> 2.1.0'
 
   s.add_development_dependency 'rake', '~> 10.4'
   s.add_development_dependency 'rspec', '~> 3.0'
```

Comment:
Should add a post install message to notify users of the library update and CLI removal